cpp/src/io/parquet/chunk_dict.cu

```diff
@@ -138,8 +190,8 @@ CUDF_KERNEL void __launch_bounds__(block_size)
     size_type is_unique      = 0;
     size_type uniq_elem_size = 0;
     if (is_valid) {
-      is_unique =
-        type_dispatcher(data_col.type(), map_insert_fn{hash_map_mutable}, data_col, val_idx);
+      auto const is_unique =
```

Comment:
Need to use the above `is_unique` at L190 here and not declare a new one! *sigh*